project/ws/app/src/lib/routes/home/routes/approvals/approval-pending/approval-pending.component.html

```diff
@@ -67,14 +67,14 @@
           <ws-app-searchuser (handleApiData)="onSearch($event)" [isApprovals]="true" (handleapproveAll)="onApproveALL()"
             [showApproveALL]="false" [showBulkUpdate]="false"></ws-app-searchuser>
         </div>
-        <ng-container *ngIf="transfersData && transfersData.length > 0">
+        <ng-container *ngIf="totaltransfersRecords > 0">
           <ws-widget-user-card [currentFilter]="currentFilter" [usersData]="transfersData" [isApprovals]="true"
             [totalRecords]="totaltransfersRecords" [tabChangeIndex]="pageIndex"
             (searchByEnterKey)="onEnterkySearch($event)" (disableButton)="showButton()"
             (paginationData)="onPaginateChange($event)" (updateList)="fetchApprovals()"></ws-widget-user-card>
         </ng-container>
-        <ng-container *ngIf="!transfersData || transfersData.length === 0">
-          <p class="margin-top-xl margin-bottom-xl">No approvals</p>
+        <ng-container *ngIf="totaltransfersRecords === 0">
```

Comment:
Please these variable in camelCase like totalTransfersRecords, same for above

Reply:
fixed and pushed the change